ACHDictionary.go

```diff
@@ -84,22 +87,23 @@ func (f *ACHDictionary) Read() error {
 	for f.scanner.Scan() {
 		f.line = f.scanner.Text()
 
-		if err := f.parseParticipant(); err != nil {
-			return err
+		if len(f.line) != 155 {
```

Comment:
`len(string)` can give incorrect results with UTF-8 strings. `unicode/utf8.RuneCountInString` will return the string's length.